dom.go

```diff
@@ -453,11 +468,18 @@ func Rerender(c Component) {
 	if prevRender == nil {
 		panic("vecty: Rerender invoked on Component that has never been rendered")
 	}
-	nextRender, skip := renderComponent(c, prevRender)
+	if c.Context().unmounted {
+		global.Get("console").Call("warn", "vecty: Rerender invoked on unmounted Component")
```

Comment:
I'm not sure I like this. I feel semi-strongly that this should either be a panic or non-existent.

In this situation, is the user of Vecty at fault / they made a programming mistake (panic)?

Or is this just something that can occur because of Rerender being incidentally invoked after a component is unmounted in an event handler (silent / no warning)?

Reply:
The user _may_ have made a programming mistake to produce this that they didn't intend, but I think it's most likely to occur due to a race in some evented code that they don't necessarily care about.